### PR TITLE
[DFFE-2] Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by cindyemeh**

### CODE REVIEW

**Comment by Ireti203** [comment]
You should add the screenshot of the webpage you worked on...i.e the product page

File: submissions/cindyEmeh/html/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+         <li><a href="https://www.twitter.com"> Twitter </a></li>
+         <li><a href="https://www.instagram.com"> Instagram </a></li>
+         <li><a href="https://www.freepik.com"> FreePik </a></li>
+       </ul>
+       <p>&copy;2021 Be Spectacled!</p>
+     </footer>
+     </body>
+     </html>
+</body>
+</html>
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:60]: Why do you have two closing tags for html and body?

File: submissions/cindyEmeh/html/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        </ol>
+     </section>
+
+     <section><em><b>Similar products</b></em></section>
+     <ol>
+       <li>Smart Glasses</li>
+       <li>Glass Cleaning Kit</li>
+       <li>Eyelasses Case</li>
+
+
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:46]: Remove whitespace

File: submissions/cindyEmeh/html/index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+             
+             <p><strong>Price:</strong><em>&#8358;10,000</em></p>
+             <p>Get this spectacle for your girl, if she has these features:</p>
+       <ol>
+         <li>Adventurous.</li>
+         <li>Free-spirited.</li>
+         <li>Fashionista-in-the-making.</li>
+         </ol>
+     </div>
+     <div><h3>Yum Spectacles</h3>
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:28]: I can't see the closing tag of the div you opened here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/cindyEmeh/html/index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="UTF-8">
+       <title>BE SPECTACLED!</title>
+</head>
+<body>
+  <h1>BE SPECTACLED!</h1>
+
+<h4>Be Spectacled! is your one stop to all types of spectacles that reflect the personality of your woman. 
+  <p>Oh yes, spectacles do reflect personalities! </p>
+  <p>Are you surprised to hear that?</p>
+  It does not matter if your woman is an introvert, extrovert or ambivert, we have ranges of spectacles to be selected from.</h4>
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:13]: Format this block of code properly

File: submissions/cindyEmeh/html/index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+    <section> 
+        <h2>Products</h2>
+        <div><h3>Bae Spectacles</h3>
+            <img src= "young-teen-woman-sunglasses-hat-holding-shopping-bags-her-hands-feeling-so-happiness-isolated-green-wall.jpg" width="500" height="400"  alt="Sale photo created by diana.grytsku - www.freepik.com">
+
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:19]: Remo whitespace

File: submissions/cindyEmeh/html/index.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        <div><h3>Bae Spectacles</h3>
+            <img src= "young-teen-woman-sunglasses-hat-holding-shopping-bags-her-hands-feeling-so-happiness-isolated-green-wall.jpg" width="500" height="400"  alt="Sale photo created by diana.grytsku - www.freepik.com">
+             
+             <p><strong>Price:</strong><em>&#8358;10,000</em></p>
+             <p>Get this spectacle for your girl, if she has these features:</p>
+       <ol>
+         <li>Adventurous.</li>
+         <li>Free-spirited.</li>
+         <li>Fashionista-in-the-making.</li>
+         </ol>
```
Comment (Ireti203) [inline submissions/cindyEmeh/html/index.html:26]: Format this block of code properly

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by cindyemeh** [comment]
Thanks for the review.

I am on it.

**Comment by cindyemeh** [comment]
Kindly review. Changes have been made.
Thank you.

**Comment by Ireti203** [comment]
Upload the screenshot of the web page

**Comment by Ireti203** [comment]
@cindyemeh You have made the changes rewuested on this PR. You should add, commit and push your changes to the branch.

**Comment by Ireti203** [comment]
@cindyemeh You have not made the changes requested on this PR.  You should push your changes to the 'page' branch

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**